formats/help/publisher/details.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "body"
+  ],
+  "properties": {
+    "body": {
+      "$ref": "#/definitions/multiple_content_types"
+    },
+    "external_related_links": {
```

Comment:
This is referenced in `placeholder` and `generic_with_external_related_links`. It could be added as a definition and referenced.

We haven't been very good at creating definitions for repeated patterns in the past: https://github.com/alphagov/govuk-content-schemas/pull/505